Sanity Check Install Content (#3584)
knative/docs: docs/install/README.md

# Installing Knative

!!! tip

>     If you're looking for an easy way to install a local distribution of Knative for prototyping, check out our [Getting Started guide](../getting-started/getting-started.md){_blank}

Reviewer: ```suggestion
    You can install a local distribution of Knative for development use, by following the [Getting started guide](../getting-started/getting-started.md){_blank}.
```
